packages/react/src/components/TimePicker/TimePicker.tsx

```diff
@@ -156,14 +177,14 @@ const clampTimeRange = (timeRange: ITimeRange): ITimeRange => {
   };
 };
 
-const generateDefaultTime = (increments: number, timeRange: ITimeRange | undefined) => {
-  const newDefaultTime = new Date();
+const generatebaseDate = (increments: number, timeRange: ITimeRange | undefined, defaultValue?: Date) => {
```

Comment:
nit, the casing on this should be `generateBaseDate`

Reply:
Resolved by correcting casing.